### DISCUSSION TITLE
Does HydroMT process data that is in sinusoidal grid projection (e.g. MODIS LAI)

### DISCUSSION THREAD

**Discussion by tnlim** [Deltares/hydromt#485] [Q&A] [answered]
Hi. Are we required to reproject MODIS LAI data when preparing the data catalogue for HydroMT? My study area requires multiple tiles of MODIS LAI (MCD15A3) data.

**Comment by DirkEilander** [comment, marked as answer]
Thanks for your question @tnlim. I guess you want to use this data to setup a Wflow model in combination with the HydroMT-Wflow plugin, correct? To my knowledge the data was not reprojected, but only merged in preparation of our deltares_data DataCatalog. @hboisgon can you confirm? We are in the process of improving the description of required preprocessing, see https://github.com/Deltares/hydromt/issues/356

**Reply by tnlim** [reply]
@DirkEilander Yes, that is what I am looking to do. Thank you for the information.
